solidity/contracts/bridge/Bridge.sol

```diff
@@ -123,6 +123,12 @@ contract Bridge is Governable, EcdsaWalletOwner {
 
     event MovedFundsSwept(bytes20 walletPubKeyHash, bytes32 sweepTxHash);
 
+    event MovedFundsSweepRequestTimedOut(
+        bytes20 walletPubKeyHash,
+        bytes32 movingFundsTxHash,
+        uint32 movingFundsTxOutputIndex
```

Comment:
Should we say *moving* or *moved*? I have not yet reached that part of the code but I think we require `submitMovingFundsProof` to be called first. In that case, funds are moved and await for sweeping.

Reply:
*Moving* because here we refer to the moving funds transaction submitted via `submitMovingFundsProof`. This is the transaction that produced the UTXO being subject of the moved funds sweep request.

Reply:
Makes sense!